src/harness/testcases/testdata/fss_record_0.json

```diff
@@ -21,5 +21,6 @@
         }
       }
     }
-    ]
+  ],
+  "ttc": true
```

Comment:
I think this needs to go one level higher, as discussed in today's call.

Reply:
ok, done, please take a look. We will have to add "incumbentProtectionData" tag for this.

Reply:
I think you have misread the spec; please take another look.

Also note that this change will require changing any code which uses this file.

Reply:
the "incumbentProtectionData" defined in the spec. contains the "id", "type", "deploymentParam" etc. which is the contents of the current top level in the data, without adding this "incumbentProtectionData" tag, it will not be a valid JSON. Can you please be more specific how it shall be. Agreed that we need to modify other codes using this data. Will do after this is finalized.